features-json/project_json_controller.rb

```diff
@@ -0,0 +1,16 @@
+require_relative "../shared/authenticated_controller_base"
+
+module FastlaneCI
+  # Controller for providing all data relating to projects
+  class ProjectJsonController < AuthenticatedControllerBase
```

Comment:
I think the general consensus is to use JSON in all caps for json when it comes to class naming `ProjectJsonController` -> `ProjectJSONController`

Reply:
done

Reply:
ngRX might be overkill for now. 
Observables are really cool, in theory, but great care must be taken when you start operate with actual data. I would suggest checking out: http://rxmarbles.com, it has all the operators with interactive graphical representations of what they do.
There's also a really great iOS app: https://itunes.apple.com/us/app/rxmarbles/id1087272442